projects/1.20/assets/toms-peripherals/toms_peripherals/lang/zh_cn.json

```
@@ -0,0 +1,38 @@
+{
+	"block.toms_peripherals.gpu": "图形处理器",
+	"block.toms_peripherals.monitor": "显示器",
+	"block.toms_peripherals.wdt": "超时复位计时器",
+	"block.toms_peripherals.redstone_port": "红石接口",
+	"block.toms_peripherals.keyboard": "键盘",
+	"block.toms_peripherals.keyboard_dongle": "无线键盘信号接收器",
+	
+	"item.toms_peripherals.gpu_chip": "图形处理器核心",
+	"item.toms_peripherals.gpu_chip_raw": "未经烧制的图形处理器核心",
+	"item.toms_peripherals.laser_pointer": "激光笔",
+	"item.toms_peripherals.portable_keyboard": "无线键盘",
+	
+	"itemGroup.toms_peripherals.tab": "Tom's Peripherals(汤姆的外设)",

```

建议直接把中文留下，或者不翻
比如
```suggestion
	"itemGroup.toms_peripherals.tab": "Tom's Peripherals",
```
或者
```suggestion
	"itemGroup.toms_peripherals.tab": "汤姆的外设",
```